SELab-2/Dr-Trottoir-4
# Feature/list manuals

Fixes #410.


As emerged in today's meeting, Seng is fixing the last task of this issue.


------------------------------

# How to test?

POST a manual via postman:

![image](https://user-images.githubusercontent.com/71405687/236569142-1ccd027f-1485-4abf-99b1-e49070be5839.png)

![image](https://user-images.githubusercontent.com/71405687/236569191-16213ef8-a266-4e2f-944e-97a625539ace.png)




Notice I chose for building 1.


Log in with your favorite admin account and go to data -> gebouwen. Look for Grote Markt 1, Antwerpen (this is building 1) and go to the info page.

As an admin you get to see all manuals:

![image](https://user-images.githubusercontent.com/71405687/236569644-7ff5c8b6-8972-4b0c-886f-01e7b35ab732.png)


As a syndic (`sylke@test.com`) only the most recent one:

![image](https://user-images.githubusercontent.com/71405687/236569751-160bf64e-79ad-4939-858f-e59437d2dd76.png)
